sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/RowEncoderSuite.scala

```diff
@@ -125,7 +125,7 @@ class RowEncoderSuite extends CodegenInterpretedPlanTest {
     new StructType()
       .add("mapOfIntAndString", MapType(IntegerType, StringType))
       .add("mapOfStringAndArray", MapType(StringType, arrayOfString))
-      .add("mapOfArrayAndInt", MapType(arrayOfString, IntegerType))
```

Comment:
`arrayOfString` doesn't work anymore inside map?

Reply:
That is a mistake. My bad :)